src/QMCDrivers/Crowd.h

```diff
@@ -28,8 +28,11 @@ class Crowd
 {
 public:
   using MCPWalker = MCPopulation::MCPWalker;
+  // This is getting a bit eh.
```

Comment:
Decryption please. Not scalable? Ugly?

Reply:
I'm not happy with the MCPWalker name.  But Walker isn't a safe type name to use in the code yet.

Reply:
Do we have a rule for adding comments before or after? I prefer before the code.

Reply:
There is a rule and its before. To be honest I'm not 100% sure which type alias I was commenting on I will clean